meinberlin/assets/scss/components/_filter_bar.scss

```diff
@@ -5,12 +5,31 @@
     .dropdown {
         display: inline-block;
     }
-}
 
-.filter-bar__filters {
-    float: left;
+    @media (max-width: $breakpoint) {
+        > * {
+            @include grid-same-width(2);
+        }
+
+        .dropdown {
+            display: block;
+        }
+
+        .dropdown-toggle {
+            width: 100%;
+            max-width: none;
+        }
+    }
 }
 
 .filter-bar__ordering {
-    float: right;
+    @media (min-width: $breakpoint) {
+        float: right;
+    }
+
+    @media (max-width: $breakpoint) {
+        // push to the right
+        // NOTE: needs to overwrite grid-tiles
+        @include grid-position(6);
```

Comment:
On the [development server](https://meinberlin-dev.liqd.net/projects/test/) this does not work: The ordering button should be on the right, but it is left aligned.